Dockerfile

```diff
@@ -17,11 +17,9 @@ ADD ${RESOURCE_PATH}/solr-${SOLR_VERSION}.tgz /tmp
 RUN useradd  --system --user-group $SOLR_USER  && \
     cd /tmp && \
     #install lsof dependency as does ansible installer
-    yum install lsof -y && \
-    tar -xzf solr-$SOLR_VERSION.tgz -C /opt && \
```

Comment:
@naveen-armedia Looks like the tarball is left as a .tgz rather than being unpacked... Is that intentional?

Reply:
@fabricetriboix  Yes. Since  the `ADD` instruction extract the tgz file to the destination.